src/components/dashjs-statistics/dashjs-statistics.tsx

```diff
@@ -1,296 +1,341 @@
-import { Component, Host, h, Prop, Watch, Element, State} from '@stencil/core';
+import { Component, Host, h, Prop, Watch, Element, State, Listen } from '@stencil/core';
 import * as chartjs from 'chart.js';
 const { Chart } = chartjs.default.Chart;
 
-
 @Component({
   tag: 'dashjs-statistics',
   styleUrl: 'dashjs-statistics.css',
   shadow: false,
 })
-
 export class DashjsStatistics {
+  @Element()
```

Comment:
Can you split this up into a general statistics component which reuses what both video and audio are using (like displaying the graphs etc.) and then using it in here. Because this component is getting too big and is mixing multiple functionalities